### ISSUE TITLE
No cron in nginx-proxy ? bizare

### ISSUE THREAD

**Issue by tcarecolin** [nginx-proxy/nginx-proxy#1523]
## Question:
Hello,
excuse me for opening an issue for that, but https://groups.google.com/forum/#!forum/nginx-proxy <- not found, i tried but no way to find this group in google (i have a valid google account).

## is this a Bug or Feature ?

I have a question regarding cron jobs, i am using jwilder's Dockerfile to build my own nginx image, i need tu use cron jobs but the command crontab is not found on the nginx image, despite that apparently there is a /etc/cron.daily folder with some scripts in it. 

### TLDR:
no file **/etc/crontab**, no **crontab** command, how should i manage to execute my hourly/daily/weekly stuffs ?

Thibaut
